libsolidity/formal/SymbolicVariables.h

```diff
@@ -40,6 +40,11 @@ class SymbolicVariable
 		std::string const& _uniqueName,
 		smt::SolverInterface& _interface
 	);
+	SymbolicVariable(
+		smt::SortPointer _sort,
+		std::string const& _uniqueName,
```

Comment:
Just as a side-remark: In both constructors here you should actually take the ``string`` by value and move it into the member. Can happen in a separate PR, though.

Reply:
I'll do it on this one, just a sec

Reply:
If you do it, I'd do it for both at once ;-).

Reply:
Updated

Reply:
Not only both constructors, but all the ones of the derived classes ;)